docs/asciidoc/modules/ROOT/pages/cypher-execution/parallel.adoc

```diff
@@ -19,4 +19,83 @@ include::example$generated-documentation/apoc.cypher.parallel.adoc[]
 include::example$generated-documentation/apoc.cypher.parallel2.adoc[]
 include::example$generated-documentation/apoc.cypher.mapParallel.adoc[]
 include::example$generated-documentation/apoc.cypher.mapParallel2.adoc[]
-|===
\ No newline at end of file
+|===
+
+=== apoc.cypher.parallel
+
+Given this dataset:
+
+[source,cypher]
+----
+UNWIND range(0, 9999) as idx CREATE (:Person {name: toString(idx)})
+----
+
+we can execute parallel statements through (:Person) nodes with this procedure:
+
+[source,cypher]
+----
+MATCH (p:Person) WITH collect(p) as people
+CALL apoc.cypher.parallel('RETURN a.name + t as title', {a: people, t: $suffix}, 'a')
+YIELD value RETURN value.title as title
+----
+
+In the above query, we passed a map as a second parameter and a string from the previous map as a third parameter.
+The value with key 'a' will be the the list to cycle in parallel.
+Note that it is not needed to pass parameter in `$suffix` because, under the hood, the procedure will prepend them in the query WITH  $`parameterName` as `parameterName`. So in this case, WITH  $`a` as `a`,  $`t` as `t`.
```

Comment:
@vga91, I tweaked the language just a bit, but not sure I understand the procedure correctly. Does that text look right?

Reply:
@JMHReif Thanks for the correction :) 
I mean that I can pass the values (in this case `a` and `t`) either as parameters, so `$a` and `$t` or as `a` and` t`. I just pushed an edit, I hope it is clearer.